std_msgs/Header header

geometry_msgs/Pose[] bars
std_msgs/ColorRGBA[] colors